scripts/prepare/datadog.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -eu -o pipefail
+
+cat > $1/.env <<EOF
+$(gcloud secrets versions access --secret fixie-examples_datadog_env latest)
```

Comment:
Agents in this repo are deployed/public, but I don't think it makes sense for this agent to be deployed as such since it's bound to a single datadog account?

Reply:
Hmm. Good point. For now I did a workaround where the prepare script exits with a non-zero exit code, and I modified the deploy.sh script to skip Agents where this happens.